generators/app/utility.js

```diff
@@ -692,6 +692,12 @@ function findQueue(name, account, teamProject, token, callback) {
    });
 }
 
+function kubeDeployment(target){
+   let kube = (target === 'acs' || target === 'aks' ? target : undefined);
```

Comment:
I think this is a bit confusing and potentially hard to maintain going forward. Consider being a bit more verbose.